RELEASING.md

```diff
@@ -350,6 +355,55 @@ gh release create "sdk/nodejs/${NODEJS_SDK_VERSION:?must be set}" \
 
 
 
+## Elixir SDK ⏱ `5mins`
+
+- [ ] ⚠️ Ensure that all SDKs have the same Engine version
+
+> **Warning**
+>
+> If we publish one SDK with an updated Engine version, we **must** do the same
+> for all other SDKs. This is important as currently our automatic provisioning
+> code enforces the existence of a single Engine running at a time. Users will
+> not be able to use multiple SDKs at the same time if the Engine version that
+> they reference differs.
+
+- [ ] Ensure that all checks are green ✅ for the `<SDK_GIT_SHA>` on the `main`
+  branch that you are about to release. This will usually be the commit that
+  bumps the Engine version, the one that you merged earlier.
+- [ ] Tag & publish:
+
+
+```console
+# e.g. cd sdk/elixir && export ELIXIR_SDK_VERSION=$(changie latest) && cd ../..
+# git show --summary
+# e.g. export SDK_GIT_SHA=92e7a4b1d23e0f4bb67f38cf0cbbbc5e82298e4e
+git tag "sdk/elixir/${ELIXIR_SDK_VERSION:?must be set}" "${SDK_GIT_SHA:?must be set}"
+git push origin sdk/elixir/${ELIXIR_SDK_VERSION}
+```
+
+This will trigger the [`Publish Elixir SDK`
+workflow](https://github.com/dagger/dagger/actions/workflows/publish-sdk-elixir.yml)
```

Comment:
I just realize that elixir workflows use `.yaml` instead of `.yml`. 🤦 I submit a fix in https://github.com/dagger/dagger/pull/5607 to fix filename extension.

Reply:
Merged 🚀